lib/utils/validateTypes.js

```diff
@@ -36,9 +36,43 @@ function isString(value) {
 	return typeof value === 'string' || value instanceof String;
 }
 
+/**
+ * Assert that the value is truthy.
+ * @param {unknown} value
+ * @returns {asserts value}
+ */
+function assert(value) {
+	// eslint-disable-next-line no-console
+	console.assert(value);
```

Comment:
[note] [`console.assert()`](https://developer.mozilla.org/en-US/docs/Web/API/console/assert) is available both on Node.js and modern browsers.